geoportal/c2cgeoportal_geoportal/scaffolds/update/+dot+upgrade.yaml_tmpl

```diff
@@ -25,7 +25,6 @@ default_project_file:
     - \.circleci/config\.yml
     - \.travis\.yml
     - qgisserver/geomapfish\.yaml
-    - build
```

Comment:
I do not understand everything here, what is the effect of having a file in `default_project_file`.
Should be good to have a short description for each list and a deterministic order for paths.

geoportal_{{package}}_geoportal/static/.* => geoportal/{{package}}_geoportal/static/.*
_ => /

Reply:
Updated thanks :-)